File: test/com/leetcode/array/TwoSumTest.java
package com.leetcode.array;


import static org.junit.jupiter.api.Assertions.assertEquals;

import com.leetcode.array.sum.TwoSum;
import org.junit.jupiter.api.Test;
/**
 * packageName    : com.leetcode.easy
 * fileName       : TwoSumTest
 * author         : Jihun Park
 * date           : 2022/01/05
 * description    :
 * ===========================================================
 * DATE              AUTHOR             NOTE
 * -----------------------------------------------------------
 * 2022/01/05        Jihun Park       최초 생성
 */
class TwoSumTest {
    TwoSum twoSum = new TwoSum();
    @Test
    void twoSum() {
        assertEquals(9,twoSum.twoSum(new int[]{2,7,11,15},9));
    }
}

File: test/com/leetcode/TwoSumTest.kt
package com.leetcode

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

/**
 * Created by jihun.park@navercorp.com on 2022-09-07.
 */
internal class TwoSumTest {

    @Test
    fun twoSum() {
        val twoSum = TwoSum()
        assertEquals(intArrayOf(0,1), twoSum.twoSum(intArrayOf(2, 7, 11, 15), 9))
    }
}